README.md

```diff
@@ -22,73 +21,36 @@ This is the source for [www.php-fig.org][php-fig.org]. It is a static site gener
     [edit]:  https://github.com/blog/905-edit-like-an-ace
     [fig-standards]: https://github.com/php-fig/fig-standards
 
-
 ## Clone
 Note the `--recursive` flag to clone the submodule too.
 ```bash
 git clone --recursive https://github.com/php-fig/www.php-fig.org
 ```
 
+## Install and run
+The project leverages `make` and [Docker Compose](https://docs.docker.com/compose/) to automate the local environment, so they are required.
 
-## Install
-
-While the project was ported from [Jekyll][jekyll] to [Sculpin][sculpin] to use PHP, some Ruby dependencies are still present. This will probably change in the future.
-
-```bash
-bin/install.sh
-```
-
-will install both PHP Composer packages and Ruby bundled gems.
-
-  [jekyll]: https://github.com/mojombo/jekyll
-  [sculpin]: https://sculpin.io
-
+Once you have them installed, you can simply type `make` and the whole site will be built and put on "watch mode", for both PHP and SASS/CSS.
+You'll then be able to access the site at `localhost:8000`.
 
-## Build
+Additionally, you can run `make html-proofer` to run the same checks that are run during the deploy process. Look into `Makefile` for additional useful targets. 
 
-```bash
-bin/build.sh
-```
-
-will compile the sources into `output_dev`.
-
-
-## Run
-
-```bash
-vendor/bin/sculpin serve
+Note that:
+- you can open a shell inside the PHP container with `make shell`
+- inside the PHP container, relevant executables are in the global `PATH`
+- the container is executed with user and group with id `1000` by default, which will likely match your main host user. You can change the id via the `UID` and `GID` build-time variables; you can configure those locally creating a `docker-compose.override.yml` (which is ignored by Git) containing:
+```yaml
+version: '3.5'
+
+services:
+  php:
+    build:
+      args:
+        # your custom IDs
+        UID: 1234
+        GID: 5678
```

Comment:
you can add in the makefile

```
WWW_DATA_USER_ID := $$(id -u)
WWW_DATA_USER_GID := $$(id -g)
```
to automatically get the right uid and gid, so the user does not have to manually adjust them

Reply:
This is a nice suggestion, but it would break for any direct usage of Docker Compose without the makefile...